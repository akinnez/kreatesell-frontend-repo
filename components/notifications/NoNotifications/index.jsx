import Image from 'next/image';
import Clipboard from 'public/images/clipboards.png';
import styles from './index.module.scss';

const NoNotifications = ({width = 70, height = 70}) => (
	<div className={styles.no__notifications}>
		<Image src={Clipboard} alt="Clipboard" width={width} height={height} />
		<p>No notifications yet</p>
	</div>
);

export default NoNotifications;
