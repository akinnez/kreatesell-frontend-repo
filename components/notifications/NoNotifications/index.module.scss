@use "public/css/colors";
@use "public/css/mixins";

.no__notifications {
  @include mixins.flex(column, space-between, center);
  row-gap: 20px;

  p {
    margin: 0;
    color: colors.$black-100;
    font: {
      size: 16px;
      weight: bold;
    }
    line-height: 1.3;
  }
}
