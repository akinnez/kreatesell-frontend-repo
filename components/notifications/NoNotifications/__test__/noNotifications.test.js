import renderer from "react-test-renderer";
import NoNotifications from "..";

it("renders a snapshot of <NoNotifications />", () => {
  const el = renderer
    .create(<NoNotifications width={70} height={70} />)
    .toJSON();
  expect(el).toMatchSnapshot();
});
