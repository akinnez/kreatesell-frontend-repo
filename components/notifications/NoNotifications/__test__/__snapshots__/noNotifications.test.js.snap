// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <NoNotifications /> 1`] = `
<div
  className="no__notifications"
>
  <img
    src="[jest.find('../utils')]"
  />
  <p>
    No notifications yet
  </p>
</div>
`;
